src/router/response.rs

```diff
@@ -32,11 +35,15 @@ impl Response {
         self.body
     }
 
-    pub fn headers(&self) -> &Option<Vec<(header::HeaderName, &'static str)>> {
+    pub fn headers(&self) -> &Option<Vec<(header::HeaderName, String)>> {
```

Comment:
Same for this, it can also be `Cow` or `AsRef<str>` to make use of both.

Reply:
Failed to do this for multiple headers as `Cow` does not implement `Copy`. Conversion from `Into<Cow<'static, str>>` is not possible.